components/admin_console/data_grid/data_grid.tsx

```diff
@@ -278,7 +281,7 @@ class DataGrid extends React.PureComponent<Props, State> {
     public render() {
         return (
             <div
-                className='DataGrid'
+                className={classNames('DataGrid', this.props.customGridClass)}
```

Comment:
Nit: prefer the use of `className` for the name here over `customClass`. Might just be a personal thing, but it lines up with the attribute that other HTML elements use. 0/5 on the change though.